packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -31,16 +31,12 @@ function findItem<T extends Item>(
   strict: boolean,
   suggestions: T[],
   value: T | T['id'],
-): T | null {
+): T | T['id'] | null {
```

Comment:
Ideally I would have liked to define a type overload for this function so that if `strict: true` is used - the input value would be `T` and `strict: false` would be `T['id']`. But that's not possible due to a TS limitation

https://github.com/microsoft/TypeScript/issues/24413